assets/js/googlesitekit/widgets/index.js

```diff
@@ -43,6 +43,27 @@ const Widgets = {
 	 */
 	components: { Widget },
 
+	/**
+	 * Default contexts for Site Kit widgets.
+	 *
+	 * @since n.e.x.t
+	 */
+	contexts,
+
+	/**
+	 * Default areas for Site Kit widgets.
+	 *
+	 * @since n.e.x.t
+	 */
+	areas,
```

Comment:
Let's not expose these defaults on the global object, we can import them directly where needed. Other plugins can register their own, and those wouldn't be available here, so we shouldn't set that precedence for now.

Reply:
Updated.